src/components/ApplicationGroup/AppGroupDetailsRoute.tsx

```diff
@@ -150,15 +156,86 @@ export default function AppGroupDetailsRoute({ isSuperAdmin }: AppGroupAdminType
         setAppListLoading(false)
     }
 
-    const openCreateGroup = (e, groupId?: string) => {
+    const handleToast = (field: string) => {
+        return (
+            toast.info(
+                <ToastBody
+                    title={`Cannot ${field} filter`}
+                    subtitle={`You can ${field} a filter with only those applications for which you have admin/manager permission.`}
+                />,
+                {
+                    className: 'devtron-toast unauthorized',
+                },
+            )
+        )
+    }
+    
+    async function getPermissionCheck(payload: CheckPermissionType, _edit?: boolean, _delete?: boolean) {
+        setLoading(true)
+        try {
+            const {result} = await appGroupPermission(envId, payload)
+            if(result && !_delete) {
+                setShowCreateGroup(true)
+            } else if(result && _delete) {
+                setIsPopupBox(true)
+                setShowDeleteGroup(true)
+            }
+        } catch (err) {
+            if(err['code'] === 403) {
+                let arrUnauthorized = []
+                err['errors'].map((errors) => {
+                    arrUnauthorized.push([...errors['userMessage']['unauthorizedApps']])
```

Comment:
you need to update this in the state to show on create page

Reply:
Updated state stored in hashmap